awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -193,7 +193,7 @@ def create_reference_data(source_dir, env, content):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('this_kind', CLOUD_PROVIDERS)
+@pytest.mark.parametrize('this_kind', [plugin_name for plugin_name in discover_available_cloud_provider_plugin_names() if plugin_name != 'constructed'])
```

Comment:
The slack suggestion can be implemented as
```suggestion
@pytest.mark.parametrize(
    'this_kind',
    [
        pytest.param(
            plugin_name,
            marks=pytest.mark.xfail(reason='Fails due to changes in #15537') if plugin_name == 'constructed' else []
        )
        for plugin_name in discover_available_cloud_provider_plugin_names()
    ],
)
```

This will surface how exactly this one subtest is failing without crashing the entire pytest session.

Reply:
This was resolved via this commit: https://github.com/ansible/awx/pull/15537/commits/705d8c081226f1dc36cf1d193e1a647d512b00dc